kclvm/ast_pretty/src/node.rs

```diff
@@ -122,22 +122,20 @@ impl<'p, 'ctx> MutSelfTypedResultWalker<'ctx> for Printer<'p> {
         self.stmts(&if_stmt.body);
         self.write_indentation(Indentation::Dedent);
         if !if_stmt.orelse.is_empty() {
-            if let ast::Stmt::If(elif_stmt) = &if_stmt.orelse[0].node {
```

Comment:
I think the current code modification is incorrect, which resulted in the test not passing. I suggest adding unit testing for any modifications. https://github.com/kcl-lang/kcl/blob/main/kclvm/ast_pretty/src/tests.rs

I think the correct way to fix this issue is to perform pattern matching on orelse, printing elif only when orelse has and only has one if stmt statement, and in other cases, it is a list of else statements

```rust
if if_stmt.orelse.len() == 1 {
    if let ast::Stmt::If(elif_stmt) = &if_stmt.orelse[0].node {
                      // Nested if statements need to be considered,
                // so `el` needs to be preceded by the current indentation.
                self.fill("el");
                self.walk_if_stmt(elif_stmt);
   } else {
                       // Nested if statements need to be considered,
                // so `el` needs to be preceded by the current indentation.
                self.fill("else:");
                self.write_newline_without_fill();
        ...
   }
} else {
                   // Nested if statements need to be considered,
                // so `el` needs to be preceded by the current indentation.
                self.fill("else:");
                self.write_newline_without_fill();
   ...
}
```

Reply:
Hey! fixed this in the latest commit, all the tests are passing now. I realised later that my fix was formatting each `if` stmt to have an `elif` stmt, my bad.